src/ui/components/ExplorerHome/ExplorerTimeline.js

```diff
@@ -30,20 +30,19 @@ const ExplorerTimeline = (props: ExplorerTimelineProps): ReactNode => {
   const height = props.height || 25;
   const viewBox = `0 0 ${width} ${height}`;
   const intervals = props.timelines.cred.length;
-  let grainAsNumber, range, grainValues;
+
+  let grainValues;
   if (grainExists) {
     const grain = props.timelines.grain || [];
-    grainAsNumber = grain.map((g) => {
+    const grainAsNumber = grain.map((g) => {
       return Number(g);
     });
```

Comment:
```suggestion
    const grainAsNumber = grain.map(Number);
```
because `Number()` is a function, you can actually shorten the map to this!